src/main/java/org/embulk/input/zendesk/utils/ZendeskUtils.java

```diff
@@ -0,0 +1,167 @@
+package org.embulk.input.zendesk.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Charsets;
+import org.embulk.input.zendesk.models.Target;
+
+import org.embulk.spi.Column;
+import org.embulk.spi.ColumnVisitor;
+
+import org.embulk.spi.Exec;
+import org.embulk.spi.PageBuilder;
+import org.embulk.spi.Schema;
+import org.embulk.spi.json.JsonParser;
+import org.embulk.spi.time.Timestamp;
+import org.slf4j.Logger;
+
+import java.util.Base64;
+
+public class ZendeskUtils
+{
+    private static final Logger logger = Exec.getLogger(ZendeskUtils.class);
+
+    private ZendeskUtils()
+    {}
+
+    public static boolean isSupportIncremental(final Target target)
+    {
+        return !Target.TICKET_FORMS.equals(target)
```

Comment:
minor: `==` is okay with enum comparison

Reply:
keep old code

Reply:
The purpose of that is to compare the equality, I mean with enum you can use `==`, it has the same meaning as `equals`.

Reply:
According to: https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals, `equals` and `==` behave the same with enum, although `==` has few more advantages (e.g. null safety) but I'm also fine with `equals` for now.